### ISSUE TITLE
Invalid cubic curve rendering

### ISSUE THREAD

**Issue by RazrFalcon** [jrmuizel/raqote#42]
```
[
        MoveTo(
            (30.0,40.0)
        ),
        CubicTo(
            (16.0,137.0),
            (171.0,45.0),
            (180.0,155.0)
        )
]
```

![out](https://user-images.githubusercontent.com/725494/57721578-39945700-768d-11e9-92ce-66151ec1df97.png)

**Comment by jrmuizel** [comment]
This looks like a poor flattening from `lyon`. In the mean time I've upped the tolerance of for flattening.

**Comment by jrmuizel** [comment]
https://github.com/jrmuizel/raqote/commit/6eb1b6e996b430eb39a6d9e87c8fa8b4be7285ac

**Comment by jrmuizel** [comment]
@nical does this screenshot match how you'd expect this curve to be flattened?

**Comment by nical** [comment]
No that's pretty awful. I filed https://github.com/nical/lyon/issues/467.

**Comment by nical** [comment]
Fixed in [lyon_geom 0.12.5](https://crates.io/crates/lyon_geom/0.12.5).
